metrics/ter/ter.py

```diff
@@ -118,16 +174,22 @@ def _compute(
         predictions,
         references,
         normalized: bool = False,
-        no_punct: bool = False,
-        asian_support: bool = False,
-        case_sensitive: bool = False,
+        ignore_punct: bool = False,
+        support_zh_ja_chars: bool = False,
+        ignore_case: bool = False,
```

Comment:
Is it fine to have `ignore_case` to `False` here ? It used to ignore casing by default

Reply:
I changed it because it seemed more consistent both with other metrics and with other inputs in this metric (in that, by default, the inputs are compared directly with no change to their form) if `ignore_case` was `False` by default. That being said, I don't think it's a big deal to keep it the way it was either, with `ignore_case` as `True` by default. Do you have a preference either way?

Reply:
We can keep the default defined by the original library, i.e. `case_sensitive=False`. It seems common to do so, e.g. other similar metrics like WER or CER are not case sensitive either.